<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addLine</class>
 <widget class="QDialog" name="addLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Order Line</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(203, 235, 255);</string>
  </property>
  <widget class="QLabel" name="bookTitleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>40</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Book:</string>
   </property>
  </widget>
  <widget class="QLabel" name="quantityLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>25</y>
     <width>65</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>24</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(26, 140, 255);
font: 10pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(26, 140, 255);
font: 10pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="bookTitleComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantityLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="invalidStock">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Sorry, there is not enough of this stock available</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rejectButton</sender>
   <signal>clicked()</signal>
   <receiver>addLine</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>addLine</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
